great_expectations/cli/docs.py

```diff
@@ -65,8 +72,62 @@ def docs_list(directory):
     else:
         list_intro_string = _build_intro_string(docs_sites_strings)
         cli_message_list(docs_sites_strings, list_intro_string)
-
-    send_usage_message(data_context=context, event="cli.docs.list", success=True)
+        failed = False
+        send_usage_message(
+            data_context=context,
+            event="cli.docs.list",
+            success=True
+        )
+    except ge_exceptions.ConfigNotFoundError as err:
+        cli_message("<red>{}</red>".format(err.message))
+    finally:
+        if failed and context is not None:
+            send_usage_message(
+                data_context=context,
+                event="cli.docs.list",
+                success=False
+            )
+
+
+@docs.command(name="clean")
+@click.option(
+    '--directory',
+    '-d',
+    default=None,
+    help="Clean data docs"
+)
+@click.option(
+    "--site-name",
+    "-s",
+    help="The site for which to generate documentation. See data_docs section in great_expectations.yml",
+)
+@click.option(
+    "--all",
+    "-a",
+    help="The site for which to generate documentation. See data_docs section in great_expectations.yml",
```

Comment:
Update help strings to reflect delete semantics

Reply:
Resolved.